tree/20_cfe_basics/cfengine/packages.cf

```diff
@@ -257,6 +257,22 @@ bundle common rpm_knowledge
       "rpm3_arch_regex" string => "\S+\s+(\S+).*";
 }
 
```

Comment:
could you move this bundle to ncf_lib.cf to make future sync easier?

Reply:
Merged to master, backported to 3.7.x and 3.10.x.

Thank you!